JSDemos/Demos/Charts/CustomAnnotations/description.md

```diff
@@ -1 +1,17 @@
-To customize an annotation, declare the SVG markup in the annotation [template](/Documentation/ApiReference/UI_Components/dxChart/Configuration/annotations/#template). Inside the template, you can access the annotation being customized. 
\ No newline at end of file
+Annotations are containers for images, text blocks, and custom content that display additional information about the visualized data. To create custom annotations, specify the following Chart properties.
```

Comment:
> Annotations are containers for images, text blocks, and custom content **that display additional information about** the visualized data.

Unfortunately, it's not quite clear to what this phrase is related exactly. It should be related to "annotations", however, due to a breakup with "images, text blocks, and custom content" it becomes not that obvious.

Therefore, I suggest the following change:

```suggestion
Annotations are designed to show additional information about visualized data. You can display images, text blocks, and custom content in annotations. To create custom annotations, specify the following Chart properties.
```